### DISCUSSION TITLE
Feature Request: Add the ability to use templates in the additional text field

### DISCUSSION THREAD

**Discussion by realroywalker** [Clooos/Bubble-Card#297] [Feature requests]
I have been playing around with bubble cards and it's really nice, but I came across one thing that would make it even better - the ability to be able to use templates in the additional text field of the popup card.
As an example - all my climate devices (Drayton Wiser) do not report current temperature as an entity state, it is an attribute within the climate entity therefore I can't use the 'entity state to display' to show this for each room (unless I start making lots of template sensors which is cumbersome).
If the 'Additional text' field could parse templates I could simply enter something like; `{{ state_attr('climate.wiser_office','current_temperature') }}` to have the attribute value included as needed.
It also adds some great flexibility such as chaining data (for example the current temperature and target temperature together.

Having the same for bubble button card etc would be the ultimate winner, since as I previously mention you can then present very useful information at a glance where the entity state line appears.

**Comment by nagtegaal** [comment]
Agree...
All fields should support templating as how the mushroom_template_card does this.
This would really make the bubble card *a lot* more flexible for power users.

I think you need to update the title of the issue/feature request to "Feature Request: Add ability to use templates in *all* text areas/fields"

**Reply by Clooos** [reply]
This is already possible but not with Jinja templates, take a look at the Templates section in the documentation.

![image](https://github.com/user-attachments/assets/48d585f3-29c3-4338-9d18-2b0eb5328f06)
